packages/twenty-front/src/modules/favorites/components/CurrentWorkspaceMemberFavoritesFolder.tsx

```diff
@@ -0,0 +1,206 @@
+  }
+`;
+
+const StyledNavigationDrawerItem = styled(NavigationDrawerItem)`
+  :active {
+    cursor: grabbing;
+
+    .fav-avatar:hover {
+      cursor: grabbing;
+    }
+  }
+`;
+
+const StyledIconHeartOff = styled(IconHeartOff)<{ isMobile: boolean }>`
+  visibility: ${({ isMobile }) => (isMobile ? 'visible' : 'hidden')};
+  color: ${({ theme }) => theme.color.red};
+
+  .navigation-drawer-item:hover & {
+    visibility: visible;
+  }
+`;
+
+export const CurrentWorkspaceMemberFavoritesFolders = () => {
+  const [activeFolderId, setActiveFolderId] = useState<string | null>(null);
+  const { createFolder, favoriteFolder } = useFavoriteFolders();
+  const { deleteFavorite } = useFavorites();
+  const isMobile = useIsMobile();
+  const [folderName, setFolderName] = useState('');
+  const currentPath = useLocation().pathname;
+  const [isFavoriteFolderCreating, setIsFavoriteFolderCreating] =
+    useRecoilState(isFavoriteFolderCreatingState);
+  const theme = useTheme();
+  const currentWorkspaceMember = useRecoilValue(currentWorkspaceMemberState);
+  const { favorites, handleReorderFavorite, favoritesByFolder } =
+    useFavorites();
+  const loading = useIsPrefetchLoading();
+  const { toggleNavigationSection, isNavigationSectionOpenState } =
+    useNavigationSection('Favorites');
+  const isNavigationSectionOpen = useRecoilValue(isNavigationSectionOpenState);
+
+  const handleFolderNameChange = (value: string) => {
+    setFolderName(value);
+  };
+
+  const handleSubmitFolder = async (value: string) => {
+    const trimmedValue = value.trim();
+    if (!trimmedValue) return false;
+
+    setIsFavoriteFolderCreating(false);
+    setFolderName('');
+    await createFolder(trimmedValue);
+    return true;
+  };
+
+  const handleClickOutside = async (
+    event: MouseEvent | TouchEvent,
+    value: string,
+  ) => {
+    const trimmedValue = value.trim();
+    if (!trimmedValue) {
+      setIsFavoriteFolderCreating(false);
+      return;
+    }
+
+    setIsFavoriteFolderCreating(false);
+    setFolderName('');
+    await createFolder(trimmedValue);
+  };
+
+  const handleCancel = () => {
+    setFolderName('');
+    setIsFavoriteFolderCreating(false);
+  };
+
+  const toggleNewFolder = () => {
+    setIsFavoriteFolderCreating((current) => !current);
+    setFolderName('');
+  };
+
+  if (loading && isDefined(currentWorkspaceMember)) {
+    return <FavoritesSkeletonLoader />;
+  }
+
+  const currentWorkspaceMemberFavorites = favorites.filter(
+    (favorite) => favorite.workspaceMemberId === currentWorkspaceMember?.id,
+  );
+
+  if (
+    (!currentWorkspaceMemberFavorites ||
+      currentWorkspaceMemberFavorites.length === 0) &&
+    (!favoriteFolder || favoriteFolder.length === 0) &&
+    !isFavoriteFolderCreating
+  ) {
+    return <></>;
+  }
+
+  const unorganisedFavorites = currentWorkspaceMemberFavorites.filter(
+    (favorite) => !favorite.favoriteFolderId,
+  );
+  const isGroup = currentWorkspaceMemberFavorites.length > 1;
```

Comment:
I think it was there before but I don't really understand what that isGroup is for and why it's named this way?

Reply:
Looking at the code, isGroup is a styling prop used in the NavigationDrawerItemsCollapsedContainer component when the navigation drawer is collapsed (!isExpanded):

isGroup={false}: item gets width: 24px
isGroup={true}: item gets width: 24px plus a lighter transparent background, matching border, and border radius